xbmc/utils/FileUtils.cpp

```diff
@@ -226,3 +227,61 @@ CDateTime CFileUtils::GetModificationDate(const std::string& strFileNameAndPath,
   }
   return dateAdded;
 }
+
+bool CFileUtils::ZebraListAccessCheck(const std::string &filePath)
+{
+  // white/black list access checks, disallow exploits
+
+  // no access to these files,
+  // this can expose user/pass of remote servers
+  static const char *blacklist_files[] = {
+    "passwords.xml",
+    "sources.xml",
+    "guisettings.xml",
+    "advancedsettings.xml",
+    NULL
+  };
+
+  for (const char **ptr = blacklist_files; *ptr; ++ptr)
+  {
+    if (filePath.find(*ptr) != std::string::npos)
+    {
+      CLog::Log(LOGDEBUG,"http access denied");
```

Comment:
Space after the comma. And I would make the log message more generic as this method is not really HTTP specific (at least its name does not indicate that).